packages/mobile/src/identity/verification.ts

```diff
@@ -100,13 +107,15 @@ export function* fetchVerificationState() {
     })
     if (timeout) {
       Logger.debug(TAG, '@fetchVerificationState', 'Token balances is null or undefined')
+      yield put(setVerificationStatus(VerificationStatus.Stopped))
       return
     }
     const isBalanceSufficientForSigRetrieval = yield select(
       isBalanceSufficientForSigRetrievalSelector
     )
     if (!isBalanceSufficientForSigRetrieval) {
       Logger.debug(TAG, '@fetchVerificationState', 'Insufficient balance for sig retrieval')
+      yield put(setVerificationStatus(VerificationStatus.Stopped))
```

Comment:
Because we have the `fianally` block, I don't think we need these here.
```suggestion
      return
    }
    const isBalanceSufficientForSigRetrieval = yield select(
      isBalanceSufficientForSigRetrievalSelector
    )
    if (!isBalanceSufficientForSigRetrieval) {
      Logger.debug(TAG, '@fetchVerificationState', 'Insufficient balance for sig retrieval')
```